src/styles/less/GamemodeStyles.less

```diff
@@ -45,14 +45,14 @@
   }
 
   // Mission Accept Btn
-.mission-explosion {
-  position: absolute;
-  // z-index: -1;
-}
-.mission-btn-txt {
-  z-index: 1;
-  margin-top: 2rem;
-}
+  .mission-explosion {
+    position: absolute;
+    // z-index: -1;
+  }
+  .mission-btn-txt {
+    z-index: 1;
+    margin-top: 2rem;
+  }
```

Comment:
I'm guessing you accidentally removed these lines and had to add them back in. Am I correct or was there some change you made that I failed to notice?

Reply:
that's right.